test/test_logging.cpp

```diff
@@ -224,13 +244,20 @@ TEST(CLASSNAME(TestLogging, RMW_IMPLEMENTATION), test_logger_severities) {
     RCUTILS_RET_INVALID_ARGUMENT,
     rcutils_logging_set_logger_level("rcutils_test_loggers", 1000));
   rcutils_reset_error();
+
+  EXPECT_EQ(RCUTILS_RET_OK, rcutils_logging_shutdown());
+  g_rcutils_logging_initialized = true;
```

Comment:
Not in this case, because the OSRF_TESTING_TOOLS_CPP_SCOPE_EXIT segment guarantees that the logging system is shutdown, and this variable is set to false again before the next tests.